### PR TITLE
Add option to specify secret name

### PR DESCRIPTION
**Pull request by rancher-max**
#954 

Also update bug template to use comments to that submitted bugs are cleaner

### CODE REVIEW

File: cli/cmd/secrets/create.go [hunk 1]
```diff
@@ -30,6 +31,13 @@ const (
 )
 
 func (s *Create) Run(ctx *clicontext.CLIContext) error {
+	name := s.N_Name
+	if name != "" {
+		r := ctx.ParseID(fmt.Sprintf("secret/%s", name))
+		if _, err := ctx.Core.Secrets(r.Namespace).Get(r.Name, metav1.GetOptions{}); err == nil {
+			fmt.Printf("Warning: %s already exists\n", r)
```
Comment (cbron) [inline cli/cmd/secrets/create.go]: Shouldn't this be `err != nil` ? Also I would check against AlreadyExists error type, this could be a diff error.

Comment (rancher-max) [reply cli/cmd/secrets/create.go]: `err == nil` in this case because it returns no error if a secret with this name exists.

Comment (cbron) [reply cli/cmd/secrets/create.go]: Ahh, right.

Comment (cbron) [reply cli/cmd/secrets/create.go]: You're printing the v1.Secret type ? Why not just name.

Comment (rancher-max) [reply cli/cmd/secrets/create.go]: I was just printing similar to how we do it elsewhere. It ends up being the name with the type prepended:
```
$ ./bin/rio secret create -n githubtoken --docker
Warning: secret/githubtoken already exists
```

**Comment by StrongMonkey** [comment]
@rancher-max actually....Can we make the name also a prompt similar to other options?

**Comment by rancher-max** [comment]
@StrongMonkey added prompt as an additional option. It will prompt if the `-name` flag is not specified, otherwise it will use the value declared in the command. It only gives a warning if using the `-name` flag with a name that already exists, otherwise it will overwrite as in current state.

**Pull request merged**